site.ts

```diff
@@ -327,7 +327,7 @@ export const createThankYouMessage = (survey) => {
     <div class="thank-you-message-container">
         <h3 class="thank-you-message-header">${survey.appearance?.thankYouMessageHeader || 'Thank you!'}</h3>
         <div class="thank-you-message-body">${survey.appearance?.thankYouMessageDescription || ''}</div>
-        <div class="footer-branding"><div>powered by ${posthogLogo} PostHog</div></div>
+        ${survey.appearance?.whiteLabel ? '' : `<div class="footer-branding"><div>powered by ${posthogLogo} PostHog</div></div>`}
```

Comment:
I think you missed this for the other types of popups 👀

Reply:
nope that's all handled with
```
if (survey.appearance?.whiteLabel) {
    surveyPopup.getElementsByClassName('footer-branding')[0].style.display = 'none'
 }
```
so that I don't have to add that line for each survey popup 😅

Reply:
ah cool